cypress/e2e/a11y.cy.js

```diff
@@ -0,0 +1,39 @@
+/// <reference types="Cypress" />
+import '../support/e2e.js'
+
+describe('A11y Test Suite', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.injectAxe()
```

Comment:
Que tal verificar que está no modo _light_ antes de começar os testes?

Assim: `cy.get('[data-theme="light"]').should('exist')`.